src/components/header.js

```diff
@@ -0,0 +1,15 @@
+import { Jumbotron } from 'reactstrap';
+import './header.css';
+
+const Header = () => {
+	return (
+		<header >
+			<Jumbotron className='m-0 bg-dark text-white text-left rounded-0'>
+				<a href='./'><h1 className='display-3 font-weight-normal'>GitHub API Playground</h1></a>

```

Comment:
Make the header smaller like the usual-sized navbars on websites. Also, please change the background to a better color.